epfast-annotations/src/main/java/com/exactpro/epfast/converter/Converter.java

```diff
@@ -0,0 +1,166 @@
+        }
+        return bigDecimal.toBigInteger();
+    }
+
+    public String decimalToString(BigDecimal bigDecimal) {
+        return bigDecimal.toString();
+    }
+
+    private boolean containsFractionalPart(BigDecimal bigDecimal) {
+        return bigDecimal.remainder(BigDecimal.ONE).compareTo(BigDecimal.ZERO) > 0;
+    }
+
+    //Integer conversions
+    public BigDecimal integerToDecimal(Integer s) {
+        return BigDecimal.valueOf(s);
+    }
+
+    public BigInteger integerToBigInteger(Integer s) {
+        return BigInteger.valueOf(s);
+    }
+
+    public Long integerToLong(Integer s) {
+        return Long.valueOf(s);
+    }
+
+    public String integerToString(Integer s) {
+        return String.valueOf(s);
+    }
+
+    // Long conversions
+    public BigDecimal longToDecimal(Long s) {
+        return BigDecimal.valueOf(s);
+    }
+
+    public BigInteger longToBigInteger(Long s) {
+        return BigInteger.valueOf(s);
+    }
+
+    public Integer longToInteger(Long s) throws Exception {
+        if (s > Integer.MAX_VALUE || s < Integer.MIN_VALUE) {
+            throw new Exception(String.format("%s cannot be converted to Integer", s));
+        }
+        return Integer.valueOf(String.valueOf(s));
+    }
+
+    public String longToString(Long s) {
+        return String.valueOf(s);
+    }
+
+    // BigInteger conversions
+    public BigDecimal bigIntegerToDecimal(BigInteger s) {
+        return new BigDecimal(s);
+    }
+
+    public Long bigIntegerToLong(BigInteger s) throws Exception {
+        long value = s.longValue();
+        if (!BigInteger.valueOf(value).equals(s)) {
+            throw new Exception(String.format("%s cannot be converted to Long", s));
+        }
+        return value;
+    }
+
+    public Integer bigIntegerToInteger(BigInteger s) throws Exception {
+        int value = s.intValue();
+        if (!BigInteger.valueOf(value).equals(s)) {
+            throw new Exception(String.format("%s cannot be converted to Integer", s));
+        }
+        return value;
+    }
+
+    public String bigIntegerToString(BigInteger s) {
+        return String.valueOf(s);
+    }
+
+    // String conversions
+    public BigDecimal stringToDecimal(String s) throws Exception {
+        try {
+            return new BigDecimal(s);
+        } catch (NumberFormatException e) {
+            throw new Exception(String.format("'%s' cannot be converted to BigDecimal", s));
+        }
+    }
+
+    public BigInteger stringToBigInteger(String s) throws Exception {
+        try {
+            return new BigInteger(s);
+        } catch (NumberFormatException e) {
+            throw new Exception(String.format("'%s' cannot be converted to BigInteger", s));
+        }
+    }
+
+    public byte[] stringToByteVector(String s) {
+        int len = s.length();
+        byte[] data = new byte[len / 2];
+        for (int i = 0; i < len; i += 2) {
+            data[i / 2] = (byte) ((Character.digit(s.charAt(i), 16) << 4)
```

Comment:
`Character.digit()` returns `-1` in case of invalid character. You need check that. Add unit test